packages/block-library/src/cover/edit.js

```diff
@@ -220,19 +285,18 @@ class CoverEdit extends Component {
 									type="number"
 									id={ inputId }
 									onChange={ ( event ) => {
-										let coverMinHeight = parseInt( event.target.value, 10 );
-										this.setState( { coverMinHeight } );
```

Comment:
removing this breaks the ability to set heights manually, this behavior (keeping the value in state) was first introduced in https://github.com/WordPress/gutenberg/pull/14785#issuecomment-482042904 by @talldan & @Jackie6 
currently, if you try to insert any value then it will directly be changed to 50 (unless you paste a value greater then 50 directly)

Reply:
Hi @senadir the input problems were fixed I also solved an existing problem where it was not possible to remove the min-height setting even if I set the input to empty some value would be set on the attribute.